images/solr-ckan/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG SOLR_MAJ_MIN_VERSION
+ARG IMAGE_REPO
+FROM ${IMAGE_REPO:-lagoon}/solr:${SOLR_MAJ_MIN_VERSION}
+ARG SOLR_MAJ_MIN_VERSION
```

Comment:
Do you need a second `SOLR_MAJ_MIN_VERSION ` argument?

Reply:
yes, as each `FROM` resets all `ARG`